db/migrate/20230502045544_add_foreign_keys_to_boarding.rb

```diff
@@ -0,0 +1,6 @@
+class AddForeignKeysToBoarding < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :boardings, :trip, index: true, foreign_key: true
+    add_reference :boardings, :boarding_point, index: true, foreign_key: true
```

Comment:
same here. check whether we are able to rollback this migration or not!

Reply:
Mentioned in the previous comment vhaia.